pkgs/top-level/python-packages.nix

```diff
@@ -15761,8 +15761,6 @@ self: super: with self; {
 
   trezor = callPackage ../development/python-modules/trezor { };
 
-  trfl = callPackage ../development/python-modules/trfl { };
```

Comment:
we need an entry in `python-aliases.nix` and ensure to check for any other packages that depend on it